src/Core/Varargs/PrintfFormatParser.cs

```diff
@@ -18,16 +18,13 @@
  */
 #endregion
```

Comment:
Yes, make `MsPrintfFormatParser` inherit from `PrintfFormatParser`, with no members. I will refactor the  base class so that "extenders" of the printf format strict can plug in. For instance, the C standard says nothing about how `printf("%S", unicode_string)` should be handled (http://www.cplusplus.com/reference/cstdio/printf/), but when using MSVC, the